packages/electrode-archetype-react-app/config/archetype.js

```diff
@@ -8,7 +8,7 @@ const utils = require("../lib/utils");
 const makeAppMode = require("../lib/app-mode");
 const userConfig = Object.assign(
   {
-    options: { reactLib: "react", karma: true }
```

Comment:
these two options must be in `config/archetype` only because they are for `npm install`.

options that are for running archetype tasks can be in https://github.com/electrode-io/electrode/blob/master/packages/electrode-archetype-react-app-dev/config/archetype.js

with support for coming from `process.env`, which can be set in `xclap.js`